client/src/util/stateManager/controlsHelpers.js

```diff
@@ -13,8 +13,8 @@ import {
 
 export function maxCategoryItems(config) {
   return (
-    config.parameters?.["max-category-items"] ??
-    globals.configDefaults.parameters["max-category-items"]
+    config.parameters?.maxCategoryItems ??
```

Comment:
I thought the camelcase change was reverted?

Reply:
shoot, I must have forgotten to revert a commit. I'm going to grep the codebase to comb for anything left behind